File: core/src/main/java/platformer/bridge/requests/ItemMasterDTO.java
package platformer.bridge.requests;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class ItemMasterDTO {
    private String itemId;
    private String name;
    private String description;
    private String rarity;
    private String imagePath;
    private int sellValue;
    private boolean stackable;
    private EquipmentDataDTO equip;
}

File: game-service/src/main/kotlin/com/games/mw/gameservice/domain/item/requests/ItemMasterDTO.kt
package com.games.mw.gameservice.domain.item.requests

data class ItemMasterDTO(
    val itemId: String,
    val name: String,
    val description: String,
    val rarity: String,
    val imagePath: String,
    val sellValue: Int,
    val stackable: Boolean,
    val equip: EquipmentDataDTO?
)